packages/crowdstrike/manifest.yml

```
@@ -1,14 +1,14 @@
 name: crowdstrike
 title: CrowdStrike Logs
-version: 1.0.3
+version: 1.0.4
 description: Collect and parse falcon logs from Crowdstrike products with Elastic Agent.
 type: integration
 format_version: 1.0.0
 license: basic
 categories: [security]
 release: ga
 conditions:
-  kibana.version: "^7.16.0"
+  kibana.version: "^7.16.1"
```

```suggestion
  kibana.version: "^7.16.0"
```

The 7.16 BC is based on https://github.com/elastic/beats/commits/f170c463ef8cfd90fd07ab55cf6618897b4ee18c so the fix is included.